src/Components/Fsi.fs

```diff
@@ -494,13 +496,27 @@ module Fsi =
             return terminal
         }
 
+    let private commentRegex =
+        Regex(@"\/{2,3}.+\S+", RegexOptions.ECMAScript)
+
+    let private multilineCommentRegex =
+        Regex(@"\(\*[\s\S]*?\*\)", RegexOptions.ECMAScript)
+
+    // TODO: add support to detect terminal size? and reformat output
+    // not defined atm in typescript vscode...
```

Comment:
not clear how to adress this for now, couldnt find in vscode type definition terminal length, maybe is to be added? i see it in proposed types def here: 
https://github.com/microsoft/vscode/blob/main/src/vscode-dts/vscode.proposed.terminalDimensions.d.ts